src/datachain/lib/dc.py

```diff
@@ -247,6 +248,8 @@ def __init__(self, *args, settings: Optional[dict] = None, **kwargs):
             **kwargs,
             indexing_column_types=File._datachain_column_types,
         )
+        send_telemetry_once("datachain_init", **kwargs)
```

Comment:
[Q] Would it simplify the code to run on import of `DataChain` instead of `init`?